doc/modules/ROOT/pages/gds-session.adoc

```diff
@@ -255,6 +255,11 @@ gds.graph.nodeProperties.stream(G, db_node_properties=["name"], node_properties=
 For a full list of the available algorithms, see the https://neo4j.com/docs/graph-data-science-client/{page-version}/api/[API reference].
 
 
+=== Limitations
+
+Topological Link Prediction algorithms are not yet supported.
```

Comment:
What's the reason for this?

Reply:
The problem is, that these GDS functions work directly over 2 nodes on the db and not the GDS graph. 

We would have to execute them on the db or fetch the relationships on demand.
As this feature is untouched since GDS 1.0, we want to wait for user requests.